azure-pipelines.yml

```diff
@@ -0,0 +1,486 @@
+    #     BuildArguments: -OfficialBuildId=$(OfficialBuildId)
+    #         -ConfigurationGroup=$(_BuildConfiguration)
+    #         -PortableBuild=true -strip-symbols
+    #         -SkipTests=true
+    #         -- /p:StabilizePackageVersion=$(IsStable)
+    #     PublishArguments: /p:PublishType=$(PublishType)
+    #       /p:TargetArchitecture=x64
+    #       /p:ConfigurationGroup=$(_BuildConfiguration)
+    #       /p:PortableBuild=true
+    #       /p:OSGroup=FreeBSD
+    #       /p:StabilizePackageVersion=$(IsStable)
+    #     _TargetArchitecture: x64
+    #   pool:
+    #     name: DotNetCore-Infra
+    #     demands:
+    #       - agent.os -equals FreeBSD
+    #   strategy:
+    #     maxParallel: 2
+    #     matrix: 
+    #       debug:
+    #         _BuildConfiguration: Debug
+    #       release:
+    #         _BuildConfiguration: Release
+
+# Platform: Windows
+- template: /eng/jobs/windows-build.yml
+  parameters:
+    Jobs:
+    # Windows x64
+    - job:
+      displayName: Windows_x64
+      agentOS: Windows
+      variables:
+          _AdditionalMSBuildArguments: "/p:SkipTests=$(SkipTests)  /p:StabilizePackageVersion=$(IsStable)"
+          _BuildFullPlatformManifest: true
+          _PortableBuild: true
+          _PublishRidAgnosticPackages: true
+          _TargetArchitecture: x64
+      pool:
+        # For public or PR jobs, use the hosted pool.  For internal jobs use the internal pool.
+        # Will eventually change this to two BYOC pools.
+        ${{ if eq(variables['System.TeamProject'], 'public') }}:
+          name: NetCorePublic-Int-Pool
+          queue: Windows.10.Amd64.Open
+        ${{ if ne(variables['System.TeamProject'], 'public') }}:
+          name: dotnet-internal-temp
+      strategy:
+        maxParallel: 2
+        matrix: 
+          debug:
+            _BuildConfiguration: Debug
+          release:
+            _BuildConfiguration: Release
+
+    # Windows x86
+    - job:
+      displayName: Windows_x86
+      agentOS: Windows
+      variables:
+          _AdditionalMSBuildArguments: "/p:SkipTests=$(SkipTests)  /p:StabilizePackageVersion=$(IsStable)"
+          _BuildFullPlatformManifest: false
+          _PortableBuild: true
+          _PublishRidAgnosticPackages: false
+          _TargetArchitecture: x86
+      pool:
+        ${{ if eq(variables['System.TeamProject'], 'public') }}:
+          name: NetCorePublic-Int-Pool
+          queue: Windows.10.Amd64.Open
+        ${{ if ne(variables['System.TeamProject'], 'public') }}:
+          name: dotnet-internal-temp
+      strategy:
+        maxParallel: 2
+        matrix: 
+          debug:
+            _BuildConfiguration: Debug
+          release:
+            _BuildConfiguration: Release
+
+    # Windows Arm64
+    - job:
+      displayName: Windows_Arm64
+      agentOS: Windows
+      variables:
+        _AdditionalMSBuildArguments: "/p:SkipTests=true  /p:StabilizePackageVersion=$(IsStable)"
```

Comment:
nit: there's a double space, and i'm not certain that you need to `"` this

Reply:
Is there a tool that you use to check for double space ? :) . Will remove double space and double quotes too.